runtime/src/main/kotlin/kotlin/native/internal/NativePtr.kt

```diff
@@ -12,6 +12,7 @@ external fun getNativeNullPtr(): NativePtr
 
 class NativePtr @PublishedApi internal constructor(private val value: NonNullNativePtr?) {
     companion object {
+        // TODO: make it properly precreated, maybe use an intrinsic for that.
```

Comment:
`getNativeNullPtr()` _is_ an intrinsic already.

Reply:
We need to come up with the way to make native NULL easily foldable.